api/config/routes.rb

```diff
@@ -5,6 +5,7 @@
     resources :regional_centers, only: :index
     resources :holidays, only: %i[index create update destroy]
     resources :service_specifications, only: %i[index create update]
+    resources :expense_sheets, only: %i[index show create update destroy]
```

Comment:
These are all possible curd operations, right?

Reply:
Oh, that's right. My bad!